Template/Template/PersonDataViewController.swift

```diff
@@ -3,10 +3,247 @@
 
 import UIKit
 
+protocol SizeDelegate: AnyObject {
+    func transfer(size: Int)
+}
+
 /// контроллер с данными человека
-class PersonDataViewController: UIViewController {
-//    override func viewDidLoad() {
-//        super.viewDidLoad()
-//
-//    }
+final class PersonDataViewController: UIViewController {
+    // MARK: - Constants
+
+    /// перечисление с плейсхолдерами
+    enum Constants {
+        static let nameText = "Имя"
+        static let lastnameText = "Фамилия"
+        static let phoneText = "Номер телефона"
+        static let sizeText = "Размер ноги"
+        static let birthdayText = "Дата рождения"
+        static let mailText = "Почта"
+        static let startTopAnchor = 113
+        static let textFiledSpace = 54
+        static let maxNumberCount = 18
+        static let saveButtonColor = UIColor(red: 225.0 / 255.0, green: 24.0 / 255.0, blue: 131.0 / 255.0, alpha: 1)
+        static let backImage = UIImage(systemName: "chevron.left")
+    }
+
+    // MARK: - Visual Components
+
+    private let nameTextFiled = PersonDataTextFiled(placeholder: Constants.nameText)
+    private let lastnameTextFiled = PersonDataTextFiled(placeholder: Constants.lastnameText)
+    private let phoneTextFiled = PersonDataTextFiled(placeholder: Constants.phoneText)
+    private let footSizeFiled = PersonDataTextFiled(placeholder: Constants.sizeText)
+    private let birthdayTextFiled = PersonDataTextFiled(placeholder: Constants.birthdayText)
+    private let mailTextFiled = PersonDataTextFiled(placeholder: Constants.mailText)
+    private let saveButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle("Сохранить", for: .normal)
+        button.titleLabel?.font = Fonts.verdanaBold16
+        button.tintColor = .white
+        button.backgroundColor = Constants.saveButtonColor
+        button.layer.cornerRadius = 12
+        button.layer.shadowColor = UIColor.black.cgColor
+        button.layer.shadowOffset = CGSize(width: 0, height: 5)
+        button.layer.shadowRadius = 3
+        button.layer.shadowOpacity = 0.5
+        button.isHidden = true
+        return button
+    }()
+
+    private let datePicker = UIDatePicker()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setTextFileds()
+        setView()
+        setSaveButton()
+        setBackButtonItem()
```

Comment:
По хорошему во viewDidLoad должно быть не больше 1-2 функций Можно эти 4 функции в одну
```suggestion
        configurateUI()
```